app/src/main/res/layout/fragment_thread_list.xml

```diff
@@ -19,166 +19,203 @@
-        app:layout_constraintBottom_toTopOf="@id/mailboxName"
-        app:layout_constraintEnd_toStartOf="@id/userAvatar"
-        app:layout_constraintStart_toEndOf="@id/menuDrawerButton"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <com.google.android.material.card.MaterialCardView
-        android:id="@+id/userAvatar"
-        android:layout_width="32dp"
-        android:layout_height="32dp"
-        android:layout_marginEnd="@dimen/marginStandardMedium"
-        app:cardCornerRadius="100dp"
-        app:layout_constraintBottom_toBottomOf="@+id/searchViewCard"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="@+id/searchViewCard">
-
-        <ImageView
-            android:id="@+id/userAvatarImage"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:contentDescription="@string/titleMyAccount"
-            android:src="@drawable/placeholder"
-            tools:src="@tools:sample/avatars" />
-
-    </com.google.android.material.card.MaterialCardView>
-
-    <TextView
-        android:id="@+id/mailboxName"
-        style="@style/H1"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/marginLarge"
-        android:layout_marginTop="@dimen/marginStandard"
-        android:text="@string/inboxFolder"
-        app:layout_constraintBottom_toTopOf="@id/swipeRefreshLayout"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/searchViewCard"
-        tools:text="Boite de Reception" />
-
-    <!-- TODO: multiselect -->
-    <!--        <com.google.android.material.button.MaterialButton-->
-    <!--            android:id="@+id/openMultiselectButton"-->
-    <!--            style="@style/TextButton"-->
-    <!--            android:layout_width="wrap_content"-->
-    <!--            android:layout_height="wrap_content"-->
-    <!--            android:text="Modifier"-->
-    <!--            app:layout_constraintEnd_toEndOf="parent"-->
-    <!--            app:layout_constraintTop_toBottomOf="@id/searchViewCard" />-->
-
-    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-        android:id="@+id/swipeRefreshLayout"
-        android:layout_width="match_parent"
         android:layout_height="0dp"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior"
+        android:fitsSystemWindows="false"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/mailboxName">
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:context=".ui.main.thread.ThreadListFragment"
+        tools:openDrawer="start">
 
         <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Move this layout to another included file, for example `fragment_thread_list_content` or `content_fragment_thread_list` 🤷🏾‍♂️